src/webui/www/public/scripts/mocha-init.js

```diff
@@ -35,6 +35,24 @@ var pauseFN = function() {};
 
 initializeWindows = function() {
 
+    saveWindowSize = function(windowId) {
+        var width = $(windowId).getSize().x;
+        var height = $(windowId).getSize().y;
+
+        localStorage.setItem('window_' + windowId + '_width', width);
+        localStorage.setItem('window_' + windowId + '_height', height);
+    };
+
+    getWindowSavedWidth = function(windowId, defaultValue) {
```

Comment:
Why do you save Size, but restore Width and Height separatelly?

Reply:
The width + height can be saved in a single call, but they're set [separately](https://github.com/Piccirello/qBittorrent/blob/a02fd91d7b7494bb877532474053bc417c9f82b5/src/webui/www/public/scripts/mocha-init.js#L77,L78).

Reply:
OK, if it's really inconsistency in used classes.

Reply:
I've modified the functions to be more consistent.

Reply:
I see no reason to split `saveWindowSize` into two functions.
I would merge getWindowWidth/Height into one function, but it will be inconvenient due to inconsistency of used code. So I would leave it as it was. @Chocobo1?

Reply:
@Chocobo1 
Although, if you approve the current code feel free to merge it.

Reply:
>So I would leave it as it was. @Chocobo1?

IMO any other form isn't much better, I'll just accept it.

>Although, if you approve the current code feel free to merge it.

OK but we are on the brink of a new release, is it really safe to merge it now?

Reply:
Updated and squashed.

> OK but we are on the brink of a new release, is it really safe to merge it now?

The changes are pretty minimal and I've been testing for the last few weeks. I think it's ready, but I'm a bit biased in this.

Reply:
>The changes are pretty minimal and I've been testing for the last few weeks. I think it's ready, but I'm a bit biased in this.

Based on previous disastrous releases, personally I rather not risk at all.
I'm thinking of a coarse releasing procedure as follow:
1. Normally we are free to merge
2. After @sledgehammer999 notify us he is ready for a new release (e.g. #8280), we should only accept 1. critical bug fixes. 2. fixes to the PRs merged in step 1.
  New features should defer to the next version.

At least I think this will minimize untested bugs exposed to users.